src/e2e/java/teammates/e2e/util/LNPTestData.java

```diff
@@ -67,8 +67,8 @@ public DataBundle generateJsonData() {
         dataBundle.courses = generateCourses();
         dataBundle.instructors = generateInstructors();
         dataBundle.students = generateStudents();
-        dataBundle.feedbackSessions = generateFeedbackSessions();
         dataBundle.feedbackQuestions = generateFeedbackQuestions();
+        dataBundle.feedbackSessions = generateFeedbackSessions();
```

Comment:
Irrelevant changes?

Reply:
Reverted back the change. Leftover from when I was trying out ways to obtain feedback question ID.